pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -306,6 +322,48 @@ func GatherContainerImages(i *Gatherer) func() ([]record.Record, []error) {
 			continueValue = pods.Continue
 		}
 
+		// Transform map into a list for sorting.
+		type tmpImageCount struct {
```

Comment:
do not inline types like this, move this outside the func

Reply:
Fair enough, my logic with both the function and the struct were that I didn't like polluting the namespace with all these helper stuff that's only used once.